components/camel-spring-batch/src/main/java/org/apache/camel/component/spring/batch/SpringBatchEndpoint.java

```diff
@@ -83,7 +85,7 @@ protected void doStart() throws Exception {
         if (jobLauncher == null) {
             jobLauncher = resolveJobLauncher();
         }
-        if (job == null && jobName != null) {
+        if (job == null && jobName != null && jobName.compareTo("dynamic")!=0) {
```

Comment:
Hi as jobName is the path and can't be made null beacause it's mandatory, "dynamic" keyword is used to omit the lookup of the Job and allow dynamic lookup based on message.  Due to camel URI schema it's IMPOSSIBLe as you said in the ticket to make it null, so I need a keyword  to know when it should be ignored. please check the ticket and your answer.